app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- Nice work 👍 
![Capture](https://user-images.githubusercontent.com/63972670/217842260-280433e6-6c69-4b52-a350-7b7e18196a8f.JPG)

Reply:
Thanks @OLIPLICHE for your review.